SU-SWS/saa_victoria_subtheme
# Adapt 710  external link button

# READY FOR REVIEW

# Summary
- Fix the underline on external link buttons, underline always shows (should be just on hover/focus)

# Review By (Date)
- TODAY 9/18

# Urgency
- medium

# Steps to Test

1. Pull down the branch or test on https://saauserguide-test.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/text-area
2. Test underline is gone regularly, then shows correctly on hover/focus.
3. All the buttons on this page should look correct.

# Affected Projects or Products
- N/A

# Associated Issues and/or People
- JIRA ticket ADAPT-710
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
